modules/eb_widget/js/dropzonejs_eb_widget.common.js

```diff
@@ -8,31 +8,30 @@
 
   Drupal.behaviors.dropzonejsEbWidgetCommon = {
     attach: function(context) {
-      if (Drupal.dropzonejsInstances.length) {
-        for (var i = 0; i < Drupal.dropzonejsInstances.length; i++) {
-          var instance = Drupal.dropzonejsInstances[i];
-          var $form = $(Drupal.dropzonejsInstances[i].element).parents('form');
+      if (typeof drupalSettings.dropzonejs.instances !== "undefined") {
+        _.each(drupalSettings.dropzonejs.instances, function (item) {
+          var $form = $(item.instance.element).parents('form');
 
-          if (form.hasClass('dropzonejs-disable-submit')) {
-            var $submit = form.find('input.form-submit');
-            submit.attr("disabled", "true");
+          if ($form.hasClass("dropzonejs-disable-submit")) {
+            var $submit = $form.find('[type=submit]');
+            $submit.attr("disabled", "true");
 
-            instance.on("queuecomplete", function () {
-              if (instance.getRejectedFiles().length == 0) {
-                submit.removeAttr("disabled");
+            item.instance.on("queuecomplete", function () {
+              if (item.instance.getRejectedFiles().length == 0) {
+                $submit.removeAttr("disabled");
               }
               else {
-                submit.attr("disabled", "true");
+                $submit.attr("disabled", "true");
```

Comment:
Attribute *disabled* doesn't have value. So, `true` is not required here.
NOTE: If you need to set value because JS/jQuery method, set `disabled` also as value.